src/ComparisonDetails/ComparisonDetails.tsx

```diff
@@ -1,22 +1,76 @@
+// import ComparisonCard from '../ComparisonCard/ComparisonCard'
 import './ComparisonDetails.scss';
-import { Link } from 'react-router-dom';
 import { Comparison } from '../comparisonData'
 
-const ComparisonDetails = (prop: { selection?: Comparison }) => {
-  const verifyProps = () => {
-    if (prop.selection) {
-      return(
+const ComparisonDetails = (prop: { selection?: Comparison, usaStateDeaths: number, totalUSADeaths: number }) => {
 
-        <section className='main'>
-          <h2 id='page-title'>Covid-19 deaths compared to {prop.selection?.data.title}</h2>
+  // function convertDeaths(numDeaths: number) {
+  //   let deathConversion = Math.floor(numDeaths/50)
+  //   displayDeaths(deathConversion)
+  // }
+  // let deathIcons = []
+  
+  // function displayDeaths(deathConversion: number) { 
+  //   if (deathConversion <= 0) {
+  //     return deathIcons
+  //   }
+  //   deathIcons.push('X')
+  //   return displayDeaths(deathConversion - 1)
+  // }
+  
+  const compareStateDeathsToUSADeaths = () => {
+    let percentOfAllUSADeaths = prop.usaStateDeaths / prop.totalUSADeaths;
+    return (
+      <p>The number of deaths due to COVID-19 in the state of Colorado account
+        for { (percentOfAllUSADeaths * 100).toFixed(1) }% of all deaths in the United States.</p>
+    )
+  }
 
-          <p>{prop.selection?.data.title}</p>
-          <p>{prop.selection?.data.deaths}</p>
+  const calculateDeathRatio = () => {
+    if (typeof prop.selection?.data.deaths === 'number') {
+      let deathRatio = (prop.usaStateDeaths / prop.selection.data.deaths)
+      return generateComparisonText(deathRatio)
+    }
+  }
+
+  const generateComparisonText = (deathRatio: number) => {
+    if(deathRatio < 1) {
+      return(
+        <section className='comparison-text'>
+          <p>Despite the staggering number of COVID-19 deaths in 2020, COVID-19 
```

Comment:
We will need to revisit this wording to incorporate the additional information that was put into the data file (i.e. tagline)